src/cpu/gemm_convolution_utils.cpp

```diff
@@ -444,18 +444,41 @@ void im2col(const conv_gemm_conf_t &jcp, const data_type_t *__restrict im,
                             const int ow_end
                                     = (oh == last_oh) ? (last_ow + 1) : jcp.ow;
                             data_t *__restrict col_ = col_k + oh * jcp.ow - ss;
-                            if (ih < 0 || ih >= jcp.ih)
+                            if (ih < 0 || ih >= jcp.ih) {
+                                /*
                                 for (int ow = ow_begin; ow < ow_end; ow++)
                                     col_[ow] = zero_val;
-                            else {
-                                for (int ow = ow_begin; ow < ow_end; ++ow) {
+                                */
+                                memset(col_ + ow_begin, zero_val,
```

Comment:
please replace `zero_val` with 0. `zero_val` was required to avoid compiler warning about inappropriate data conversion